### ISSUE TITLE
How to add a div to the generated HTML

### ISSUE THREAD

**Issue by bclinkinbeard** [HenrikJoreteg/hjs-webpack#69]
Since React 0.14 [gives a warning when rendering to `document.body`](http://facebook.github.io/react/blog/2015/09/10/react-v0.14-rc1.html#new-helpful-warnings), I was trying to add an empty div to the HTML. The README and code seem to imply this can be done, but I'm not clear on how. In fact, the README text appears to be incomplete.

> `{html: 'your string}` and it'll add it to the

**Comment by lukekarrys** [comment]
Yeah, I'll add some better wording to the docs about that. What you want is this for your html fn in the config:

```js
html: function (context) {
  return {
    'index.html': context.defaultTemplate({html: '<div id="container"></div>'})
  }
}
```

**Comment by bclinkinbeard** [comment]
That doesn't appear to be working for me.

```js
var getConfig = require('hjs-webpack');

module.exports = getConfig({
  in: 'src/app.js',
  out: 'public',
  clearBeforeBuild: true,
  html: function (context) {
    return {
      'index.html': context.defaultTemplate({html: '<div id="container"></div>'})
    };
  }
});
```

**Comment by bclinkinbeard** [comment]
I tried debugging by adding `console.dir('incomingData.html =', incomingData.html);` as the first line of `defaultHtml()` in `html-plugin.js`, and `incomingData.html` appears to be empty.

**Comment by lukekarrys** [comment]
Hmm, thats odd. What command are you running to start webpack?

**Comment by bclinkinbeard** [comment]
`webpack-dev-server` via `npm start`

**Comment by lukekarrys** [comment]
I'm not aware of any bugs we've had with the `context.defaultTemplate` fn, but can you try updating to the latest version of `hjs-webpack` (if you dont have it already)? Also, does the `npm run build` fn appear to work correctly?

**Comment by lukekarrys** [comment]
I put together [this gist](https://gist.github.com/lukekarrys/7e338f443dc98b574f0f) which appears to be working for me. Can you try cloning and running `npm install` and `npm start` and see if you are still seeing the issue there?

**Comment by bclinkinbeard** [comment]
Thanks, I figured out the issue. I was still calling `React.render(<App/>, document.body)`, since I hadn't been able to confirm the div was being added. Turns out that was just overwriting the div somehow, making it look like the template was wrong. Updating my code to `React.render(<App/>, document.getElementById('container'));` shows that the template is in fact correct.

Thanks for the help, sorry for the false alarm.
